patches/@firebase+rules-unit-testing+3.0.3.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/node_modules/@firebase/rules-unit-testing/dist/index.cjs.js b/node_modules/@firebase/rules-unit-testing/dist/index.cjs.js
```

Comment:
Hey @Salakar - any collaboration upstream on this one? Is this upstream-able ? I didn't see a PR in firebase-js-sdk related ...